spec/takeaway_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  let(:order) { double :order }
+  let(:sms) { double :sms, deliver: nil}
+  subject(:takeaway) { described_class.new(menu) }
+
+  it 'knows an order total' do
+    takeaway.order('Lobster', 2)
+    expect(takeaway.bill).to eq(22184)
```

Comment:
Are we testing that the order can calculate the total, or are we testing that the price of two lobster is 22184? I think it's the former, because the second is state/configuration and we don't want to test that.

However, this test will fail if either the order fails to calculate the bill correctly or the price of lobster changes. That makes it a brittle test. We want to mock our dependencies so that we don't need to worry about what the price of lobster really is, just that the bill is calculated correctly.

(You've done this well in other tests)